client/src/components/Panel/Panel.tsx

```diff
@@ -0,0 +1,18 @@
+import { Box, MantineTheme } from "@mantine/core";
+import React, { ReactNode } from 'react';
+
+const styles = (theme: MantineTheme) => ({
+  maxWidth: 400,
+  background: theme.white,
+  borderRadius: '5px',
+  margin: '30px 80px',
+  padding: '80px 40px',
```

Comment:
Takie hardkodowanie marginesów to troszkę mało responsywne podejście. Ogólnie to myślałem że wykorzystamy komponent Modal z mantine do zrobienia tego widoku, wtedy ten cały komponent Panel nie byłby potrzebny